chain/chaintest/block.go

```diff
@@ -104,3 +109,249 @@ func GenerateTestExecutedBlocks(
 	}
 	return executedBlocks
 }
+
+// TxListGenerator is a function that should return a list of valid TXs of
```

Comment:
```suggestion
// TxListGenerator is a function that should return a list of valid txs of
```

can we use clearly named parameters in the function signature? We are referencing `numTxsPerBlock` here, but there's no such variable

Reply:
bc52f9185